src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -41,6 +42,17 @@ public function process(ContainerBuilder $container)
 
         if ($container->has('fragment.handler')) {
             $container->getDefinition('twig.extension.httpkernel')->addTag('twig.extension');
+
+            // inject Twig in the hinclude service if Twig is the only registered templating engine
+            if (
+                !$container->hasParameter('templating.engines')
+                || array('twig') == $container->getParameter('templating.engines')
+            ) {
+                $container->getDefinition('fragment.renderer.hinclude')
+                    ->addTag('kernel.fragment_renderer', array('alias' => 'hinclude'))
```

Comment:
As of 2.7, yes. `src/Symfony/Bundle/TwigBundle/Tests/Functional/NoTemplatingEntryTest.php` proves it. There are still one or two things that won't work (like controller->render() or the possibility to use the Symfony specific template notation), but the framework itself no works well without defining the templating entry.